### PR TITLE
Make the gadget compatible with arm64

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**

### CODE REVIEW

File: gadget.yaml [hunk 1]
```diff
@@ -1,24 +1,9 @@
...
-          - image: pc-boot.img
-      - name: BIOS Boot
-        type: DA,21686148-6449-6E6F-744E-656564454649
-        size: 1M
-        offset: 1M
-        offset-write: mbr+92
-        update:
-          edition: 2
-        content:
-          - image: pc-core.img
```
Comment (valentindavid) [inline gadget.yaml:21]: Nice. That did not work anyway.

Comment (valentindavid) [inline gadget.yaml:21]: I do not think that works on x86_64. Can we just remove it for all?

Comment (xnox) [reply gadget.yaml:21]: It does work..... it boots and prints a message saying "please boot in EFI mode" =)

**Review by xnox — APPROVED** [review]

File: gadget.yaml (hunk 1 above)
Comment (alfonsosanchezbeato) [reply gadget.yaml:21]: Why do you think it does not work? It is expected to be called for DOS style disks afaiu.

Comment (xnox) [reply gadget.yaml:21]: Actually in the current grub-recovery.conf i don't see a test for efi platform boot, which used to print a message "please boot in EFI mode" https://github.com/snapcore/snapd/blob/2b72f0457b8b8d35906784a4bfeceee1cf49bbcd/bootloader/assets/data/grub-recovery.cfg#L33

Like we used to have somewhere before.... hmmm.

Comment (valentindavid) [reply gadget.yaml:21]: Do we need all this just to print a message? I would think we could print that message is in less than 440 bytes. Maybe we can replace by a really simpler MBR.

Comment (alfonsosanchezbeato) [reply gadget.yaml:21]: In any case, that would be for another PR to branch `22`

Comment (xnox) [reply gadget.yaml:21]: Indeed BIOS boot is not available on arm64 so all of these things should be discussed elsewhere.

**Review by valentindavid — APPROVED** [review]

**Pull request merged**